operations/phlare/helm/phlare/values.yaml

```
@@ -203,3 +203,6 @@ minio:
   podAnnotations:
     phlare.grafana.com/scrape: "true"
     phlare.grafana.com/port: "9000"
+
+ingress:
+  enabled: false
```

```suggestion
  enabled: false
  # hosts:
  #   - phlare.local
```

Just to give an example, just enabling fails.